docs/source/starter/introduction.rst

```diff
@@ -12,103 +12,58 @@
 
 .. _new_project:
 
-############
-Introduction
-############
 
-**************************
-What is PyTorch Lightning?
-**************************
+####################
+Lightning in 2 Steps
+####################
 
-PyTorch Lightning provides you with the APIs required to build models, datasets, and so on. PyTorch has all you need to train your models; however, there’s much more to deep learning than attaching layers. When it comes to the actual training, there’s a lot of boilerplate code that you need to write, and if you need to scale your training/inferencing on multiple devices/machines, there’s another set of integrations you might need to do.
```

Comment:
I think now it leaves a few open questions. Like, what benefits does lightning provides over Pytorch? why not just use PyTorch. Code organization is mentioned which is good, all kinds of available integrations are also a big part of it. Now it sounds more like just another framework to me rather than a platform that has available components that can be used for more future integrations.